_drafts/circle-two-point-oh.md

```diff
@@ -0,0 +1,28 @@
+Notes from meeting in NYC:
+
+* We wanted to return early in the case of fast failure like rubocop and we
+  figured out how to hack it together in 1.0, but 2.0's workflows more closely
+  matched what we wanted to do.
+
+* start by describing the series of events.
+
+* do a gist of full config file, but also get the workflows part embedded.
+
+Random thoughts today:
+
+* don't feel like you have to explain everything about Circle CI 2.0 - it's ok
+  to link to their docs.
+
+* how do we talk about Volt externally? should i refer to it as our CMS for
+  galleries?
```

Comment:
So this is a really good question. In previous posts ([example](http://artsy.github.io/blog/2016/08/09/the-tech-behind-live-auction-integration/)) we use the repo name but explain what it is the first time we refer to it. But you can do whatever works best for your post!

Reply:
I am fine with mentioning Volt because we will probably see that in the workflow diagram/screenshot, too. We could use a brief description like our CMS for partners, a Rails UI app.